tape_archive/web_client/package.json

```diff
@@ -0,0 +1,12 @@
+{
+    "name": "@girder/tape-archive",
+    "version": "1.0.0",
+    "peerDepencencies": {
+        "@girder/core": "*"
+    },
+    "dependencies": {},
+    "girderPlugin": {
+        "name": "tape-archive",
```

Comment:
The hyphen in this name seems to be causing the problem.

For reference, the issue is that this name included in [Webpack's `output.library` property](https://webpack.js.org/configuration/output/#outputlibrary), which is configured to output the first line of the [built plugin file as `var girder_plugin_${name} =`](https://github.com/girder/girder/blob/460f598604362312ab715b64e628e2f99cffee6c/girder/web_client/grunt_tasks/build.js#L226). After this file is emitted (internally) the Uglify tool comes around to minify it, and sees `var girder_plugin_tape-archive =`, which is invalid, hence the error.

Following the practices of other Girder plugins with compound names, I think the convention seems to be to use underscore names for almost everything, except the name of the Pip package that gets published. So, the `name=` field of `setup.py` should use `tape-archive`, and everything else (variables and directory names) should use `tape_archive`.